<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-09-08T00:56:46 IST" name="Default suite" finished-at="2020-09-08T00:57:51 IST" duration-ms="64132">
    <groups>
    </groups>
    <test started-at="2020-09-08T00:56:46 IST" name="Default test" finished-at="2020-09-08T00:57:51 IST" duration-ms="64132">
      <class name="Execution.TestCaseExecution">
        <test-method is-config="true" signature="Initialize()[pri:0, instance:Execution.TestCaseExecution@2d7275fc]" started-at="2020-09-08T00:56:46 IST" name="Initialize" finished-at="2020-09-08T00:57:00 IST" duration-ms="13709" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Initialize -->
        <test-method signature="verifydashboard()[pri:0, instance:Execution.TestCaseExecution@2d7275fc]" started-at="2020-09-08T00:57:00 IST" name="verifydashboard" finished-at="2020-09-08T00:57:04 IST" duration-ms="3483" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifydashboard -->
        <test-method signature="vrflinks()[pri:1, instance:Execution.TestCaseExecution@2d7275fc]" started-at="2020-09-08T00:57:04 IST" name="vrflinks" finished-at="2020-09-08T00:57:20 IST" duration-ms="16376" status="FAIL">
          <exception class="java.net.MalformedURLException">
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException
at java.base/java.net.URL.<init>(URL.java:706)
at java.base/java.net.URL.<init>(URL.java:568)
at java.base/java.net.URL.<init>(URL.java:515)
at Utility.Verifylinks.g(Verifylinks.java:50)
at Utility.Verifylinks.linktest(Verifylinks.java:31)
at Execution.TestCaseExecution.vrflinks(TestCaseExecution.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at java.base/java.net.URL.<init>(URL.java:612)
... 33 more
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- vrflinks -->
        <test-method signature="verifycontactus()[pri:2, instance:Execution.TestCaseExecution@2d7275fc]" started-at="2020-09-08T00:57:20 IST" name="verifycontactus" finished-at="2020-09-08T00:57:50 IST" duration-ms="29468" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifycontactus -->
        <test-method is-config="true" signature="close()[pri:0, instance:Execution.TestCaseExecution@2d7275fc]" started-at="2020-09-08T00:57:50 IST" name="close" finished-at="2020-09-08T00:57:51 IST" duration-ms="1034" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- Execution.TestCaseExecution -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
